test/tools/reporter/mongodb_reporter.js

```diff
@@ -103,7 +103,7 @@ class MongoDBMochaReporter extends mocha.reporters.Spec {
       catchErr(test => this.testEnd(test))
     );
 
-    process.prependListener('SIGINT', () => this.end(true));
+    process.prependOnceListener('SIGINT', () => this.end(true));
```

Comment:
After one ctrl-c I want to be able to press it again to exit rather than entering this code again. just makes exiting the debugger a bit faster.